### PR TITLE
Update authentication info & fix links

### PR DESCRIPTION
**Pull request by noon-dawg**
- Updated the authentication section to remove unnecessary links that where not working. These links were causing issues in the website repository link-checker workflow action.
- Reworked some of the text to follow our writing style and aid clarity.

### CODE REVIEW

**Comment by ashkarpetin** [comment]
Image in preview doesn't load https://preview.redoc.ly/angry-grasshopper-5/fix-links#section/Authentication

**Comment by noon-dawg** [comment]
@ashkarpetin the images seem to work locally but not in preview

File: openapi/components/securitySchemes/JWT.yaml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 description: >
-  You can create a JSON Web Token (JWT) via our [JWT Session
-  resource](https://user-api-docs.rebilly.com/#operation/PostSigninRequest).
+  A JWT is a short lifetime token that can be assigned a specific expiration time.
+  To create a JWT session, see [JWT session resource](https://user-api-docs.rebilly.com/#operation/PostSigninRequest).
```
Comment (ashkarpetin) [inline openapi/components/securitySchemes/JWT.yaml:3]:
```suggestion
  To create a JWT session, see [JWT session resource](https://user-api-docs.rebilly.com/tag/JWT-Session#operation/PostSigninRequest).
```

**Review by VladimirBerdnik — APPROVED** [review]
LGTM

**Review by ashkarpetin — APPROVED** [review]

**Pull request merged**